files/acl.py

```diff
@@ -151,69 +145,87 @@ def split_entry(entry):
     else:
         t = None
 
-    p = normalize_permissions(p)
+    return [d, t, e, p]
 
-    return [d,t,e,p]
 
-def get_acls(module,path,follow):
+def build_entry(etype, entity, permissions=None):
+    '''Builds and returns an entry string. Does not include the permissions bit if they are not provided.'''
+    return etype + ':' + entity + (':' + permissions if permissions else '')
```

Comment:
Good call, I did not know that.

Addressed in 8c8d8c9. Does that seem good/Python enough to you? (Keep in mind that this PR is 1-year old and I haven't done any Python since then :) )